SonarQube.TeamBuild.PreProcessor/RulesetGenerator.cs

```diff
@@ -33,16 +44,32 @@ public void Generate(SonarWebService ws, string sonarProjectKey, string outputFi
             {
                 throw new ArgumentNullException("outputFilePath");
             }
-            
-            var qualityProfile = ws.GetQualityProfile(sonarProjectKey);
-            var activeRuleKeys = ws.GetActiveRuleKeys(qualityProfile);
+
+            IEnumerable<string> activeRuleKeys;
```

Comment:
it would be simpler to initialize this with Enumerable.Empty<> and remove the 2 else clauses below.